app/src/main/java/com/portfolio/tasky/entry/fragments/LoginFragment.kt

```diff
@@ -0,0 +1,67 @@
+package com.portfolio.tasky.entry.fragments
+
+import android.os.Bundle
+import android.view.Gravity
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.transition.Slide
+import com.portfolio.tasky.BaseFragment
+import com.portfolio.tasky.databinding.LayoutLoginBinding
+
+
+
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
```

Comment:
Most stuff that's not being used, could be removed and added later :D

Reply:
Done